### PR TITLE
[Check-in] Add example configurations for Keycloak Services

### PR DESCRIPTION
**Pull request by nikosev**
# Summary

- Fix formatting
- Add instructions for client credentials flow
- Create section for example OIDC client configurations
- Add configuration example for Keycloak RP
- Add references to Keycloak SP docs
- Add configuration example for Keycloak SP
- Update link to SURF dashboard
- Update link to check-in service

### CODE REVIEW

**Comment by nikosev** [comment]
Hello @gwarf ,

Regarding the [Update link to SURF dashboard](https://github.com/EGI-Federation/documentation/pull/566/commits/f9a059791978a6f74e1b6897f6b9ba637141d7f6) commit, could you please check if the instructions are still relevant?
I'm not able to login to the SURFconext dashboard to test it myself, because GRNET IdP is not available.

**Review by vardizzo-lab — APPROVED** [review]

File: content/en/users/aai/check-in/faq/_index.md [hunk 1]
```diff
@@ -31,15 +31,15 @@ credentials from a Dutch IdP in eduGAIN, you or your IdP administrators need to
 request the connection. The following steps will lead you to perform the
 connection:
 
-- Connect to [SURFconext dashboard](https://dashboard.surfconext.nl/apps)
-- Search for "EGI AAI Service provider proxy"
+- Connect to [SURFconext dashboard](https://dashboard.surfconext.nl/apps/all)
+- Search for "EGI Check-in Service"
```
Comment (gwarf) [inline content/en/users/aai/check-in/faq/_index.md:35]: I would suggest something like:
```
- Open the [SURFconext dashboard](https://dashboard.surfconext.nl/apps/all) and log in
```

Comment (gwarf) [reply content/en/users/aai/check-in/faq/_index.md:35]: `you or your IdP administrators need to request the connection` to be replaced by something like `the SURFConext administrator need to request the connection`. It's likely the only one able to do this.

Comment (gwarf) [reply content/en/users/aai/check-in/faq/_index.md]: Thanks!

**Review by gwarf — APPROVED** [review]
LGTM

**Pull request merged**